### ISSUE TITLE
Shield overlay icon appears only on tab-change on Taskbar + an idea

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#760]
_From [nanofo...@gmail.com](https://code.google.com/u/117990161848247711282/) on October 14, 2012 06:42:17_

OS version: Win7   SP1   x86 ConEmu version: 121009

Initially Admin-shield overlay icon not shown on Taskbar icon. If You have more tabs, shield appears on tab-change.

I mention that synchronize admin shield visibility to Settings/Tabs/Admin shield-Suffix radio button; or make a new checkbox "Show admin shield on Taskbar icon"

...or forget that overlay icon entirely :) This admin shield is not visible if You using only 1 tab since april, so I think not needs for people.

Thank You very much! _Steps to reproduction_ 1. Open more tabs or start a Task with more tabs - no overlay shield icon on Taskbar icon
2. Switch to another tab
3. Shield icon appears

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=760_

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on October 14, 2012 07:53:07_

Check svn

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on October 14, 2012 11:20:15_

svn updated

**Status:** Fixed

**Comment by ConEmu** [comment]
_From [nanofo...@gmail.com](https://code.google.com/u/117990161848247711282/) on October 14, 2012 22:25:11_

Sorry for comment `#2` which I sent unfinished, mistakenly and I deleted immediately :(

I checked SVN - and as I did not noticed any change, I checked source and found cbTaskbarShield. Aha (!) is there a checkbox somewhere? :D

Then I quickly found "Show shield overlay" in Settings finally :)

It's really great! Thank You so much! You are unbelievable fast!

(the full truth is that, now shields on tabs are always on - but I don't care personally :P )

Again, thanks for Your work - I really appreciate it!
